tests/create_nauru_test.py

```diff
@@ -19,7 +19,7 @@ def create_nauru_test(folder):
     project = create_example(folder, "nauru")
 
     add_new_tables(project.conn)
-    add_country_borders_to_model("Nauru", project)
+    ImportPoliticalSubdivisions(model_place="Nauru", project=project).add_country_borders(source="gadm", overwrite=True)
```

Comment:
This is a little weird.  You have a class that already does something, and immediately tells it to do some more.

It is probably worth it to initialize it and then call two methods to run what you want, as you are not returning objects but rather just running procedures